masterlist.yaml

```diff
@@ -7068,6 +7068,21 @@ plugins:
       - crc: 0xE889ADEB
         util: 'SSEEdit v4.0.2f'
 
+  - name: 'Morningstar - Minimalistic Races of Skyrim.esl'
+    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/22298' ]
+    inc:
+      - 'Imperious - Races of Skyrim.esp'
+    tag:
+      - Graphics
+      - Names
+      - SpellStats
+      - Text
+    msg: [ *includesMBBF ]
+    clean:
+      # version: 1.4.0
```

Comment:
Can you take those out, I'll include the version info in the pr message.
```suggestion
```

Reply:
👍